BraveWallet/Crypto/BuySendSwap/SwapCryptoView.swift

```diff
@@ -253,12 +253,34 @@ struct SwapCryptoView: View {
           Strings.Wallet.swapCryptoAmountTitle,
           swapTokensStore.selectedFromToken?.symbol ?? ""))
       ),
-      footer: ShortcutAmountGrid(action: { amount in
-        swapTokensStore.sellAmount = ((swapTokensStore.selectedFromTokenBalance ?? 0) * amount.rawValue)
-          .decimalExpansion(precisionAfterDecimalPoint: Int(swapTokensStore.selectedFromToken?.decimals ?? 18))
-      })
-      .listRowInsets(.zero)
-      .padding(.bottom, 8)
+      footer: VStack(spacing: 20) {
+        ShortcutAmountGrid(action: { amount in
+          swapTokensStore.sellAmount = ((swapTokensStore.selectedFromTokenBalance ?? 0) * amount.rawValue)
+            .decimalExpansion(precisionAfterDecimalPoint: Int(swapTokensStore.selectedFromToken?.decimals ?? 18))
+        })
+        Button(action: {
+          swapTokensStore.swapSelectedTokens()
+        }) {
+          HStack {
+            Spacer()
+            Image(systemName: "chevron.up.chevron.down")
```

Comment:
Replace `Image` with `Label(Strings.Wallet.swapSelectedTokens, systemImage: chevron.up.chevron.down).labelStyle(.iconOnly)` and then you can remove `accessibilityLabel(Strings.Wallet.swapSelectedTokens)`